### ISSUE TITLE
Release Basilisk, Nov. edition

### ISSUE THREAD

**Issue by wolfbeast** [MoonchildProductions/UXP#859]
- [x] Merge master to release branch
- [ ] Write release notes
- [ ] Synchronize build date with @trav90 
- [ ] Build
- [ ] Publish

Travis, suggested build date the 4th?

**Comment by trav90** [comment]
The 4th should work for me.

Regarding AV1 the release notes should probably specify that only AV1 in WebM Is supported at this time (i.e. doesn’t work on YouTube yet). Hopefully prevent a flood of questions after release on why YouTube’s AV1 playlist doesn’t work.

**Comment by wolfbeast** [comment]
Yeah I've already stated in them that it's only for WebM. I'll make an extra clarification because you're probably right that it's needed for YT users.

**Comment by trav90** [comment]
Linux tarball with all the goodies are up on the FTP.
